src/drt/src/TritonRoute.i

```diff
@@ -35,7 +35,7 @@
 
 %{
 
-#include <string>
+#include <string.h>
```

Comment:
If this is for strlen then use `<cstring>`.  Did we not need the older <string> which is for std::string?

Reply:
I use now const char * instead.

Reply:
ok so switch to cstring